### PR TITLE
FeatureUtility enable no proxy

### PR DESCRIPTION
**Pull request by jjiwooLim**
#27508 
Add no_proxy property for featureUtility. 
To set `no_proxy` using the environment variable, use `no_proxy`. 
To set it using featureUtility.properties, use `http.nonProxyHosts` (name used by jvm system properties) 
Currently in beta mode for testing purpose.

### CODE REVIEW

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#libby-bot

File: dev/com.ibm.ws.install.featureUtility_fat/bnd.bnd
Comment (bensonlatibm) [inline dev/com.ibm.ws.install.featureUtility_fat/bnd.bnd:1]: the copyright date is changed, but I don't see any other changes to the bnd.bnd file?

**Review by bensonlatibm — CHANGES_REQUESTED** [review]
looks good, except for the bnd.bnd file copyright change.

File: dev/com.ibm.ws.install.featureUtility_fat/bnd.bnd
Comment (jjiwooLim) [reply dev/com.ibm.ws.install.featureUtility_fat/bnd.bnd:1]: not sure why I updated the copyright. Changing it back

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#build

**Review by bensonlatibm — APPROVED** [review]
LGTM!

**Pull request merged**